android/gradle.properties

```diff
@@ -1,3 +1,5 @@
 org.gradle.jvmargs=-Xmx4G
 android.useAndroidX=true
 android.enableJetifier=true
+org.gradle.java.home=C:/Program Files/Eclipse Adoptium/jdk-17.0.13.11-hotspot
```

Comment:
![image](https://github.com/user-attachments/assets/b5861a7d-8980-40c6-a883-65b49dcd612a)
why did you add this java path?

Reply:
Same here,  we can ommit this file when pushing chagnes on remote. 

Please change it back